client/mifare/mifarehost.c

```diff
@@ -454,7 +462,7 @@ int mfnested(uint8_t blockNo, uint8_t keyType, uint8_t *key, uint8_t trgBlockNo,
 	memset(resultKey, 0, 6);
 	// The list may still contain several key candidates. Test each of them with mfCheckKeys
 	for (i = 0; i < statelists[0].len; i+=max_keys) {
-		PrintAndLog("Keys left to check: %d", statelists[0].len - i);
+		if (statelists[0].len > 1) PrintAndLog("Keys left to check: %d", statelists[0].len - i);
```

Comment:
This is very noisy. I propose to print only now and then instead of after every may_keys.

Reply:
Updated to every 10 seconds.

```
proxmark3> hf mf nested o 0 A a0a1a2a3a4a5 4 A
--nested. sectors: 1, block no:  0, key type:A, eml:n, dmp=n checktimeout=471 us          
--target block no:  4, target key type:A           
#db# Nonce#2: cannot get nonce that != nonce#1, continuing anyway with single nonce! ntdist=160          
uid:xxxxxxxx trgbl=4 trgkey=0          
statelist 0: length:55682 block:04 keytype:0 nt:xxxx3586 ks1:xxxxBECA
statelist 1: length:55682 block:04 keytype:0 nt:xxxx3586 ks1:xxxxBECA
Nonce 1 and 2 are the same!          
We have 55682 keys to check. This will take a very long time!          
Press button to abort.          
| 55597 keys left |  66.096 keys/sec | worst case  841.1 seconds remaining |          
| 54917 keys left |  64.787 keys/sec | worst case  847.7 seconds remaining |          
| 54237 keys left |  64.714 keys/sec | worst case  838.1 seconds remaining |          
Key found in 27.42 seconds after checking 1785 keys
          
Found valid key:xxxxxx213f04          
```